### PR TITLE
[thin-client] Do not print key/value info in log

### PR DESCRIPTION
**Pull request by majisourav99**
##  Do not print key/value info in log

Do not print key/value info as they could contain PII.
## How was this PR tested?

CI
## Does this PR introduce any user-facing changes?

- [x] No. You can skip the rest of this section.
- [ ] Yes. Make sure to explain your proposed changes and call out the behavior change.

### CODE REVIEW

File: clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java [hunk 1]
```diff
@@ -537,14 +537,11 @@ public static <T, K> T tryToDeserializeWithVerboseLogging(
...
       LOGGER.error(
-          "Caught a {}, will bubble up."
-              + "RecordDeserializer: {}\nWriter schema ID: {}\nLatest schema ID: {}\nValue (md5/hex): {}\nKey (hex): {}",
+          "Caught a {}, will bubble up." + "RecordDeserializer: {}\nWriter schema ID: {}\nLatest schema ID: {}\n",
           VeniceSerializationException.class.getSimpleName(),
           dataDeserializer.getClass().getSimpleName(),
           (writerSchemaId == -1 ? "N/A" : writerSchemaId),
-          latestSchemaId,
-          checksumHex,
-          keyHex);
```
Comment (nisargthakkar) [inline clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java:547]: When I investigated a bit, these variables were computed just for the purpose of logging. Can we remove that too?

Comment (majisourav99) [reply clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java:547]: They are computed to validate if they can be desered or not.

**Review by nisargthakkar — DISMISSED** [review]

File: clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java [hunk 2]
```diff
@@ -509,24 +509,23 @@ public static <T, K> T tryToDeserializeWithVerboseLogging(
...
       }
 
       try {
-        keyHex = Hex.encodeHexString(keySerializer.serialize(key));
+        Hex.encodeHexString(keySerializer.serialize(key));
       } catch (Exception e3) {
-        keyHex = "failed to serialize key and encode it as hex";
-        LOGGER.error("{} ...", keyHex, e3);
+        String msg = "failed to serialize key and encode it as hex";
+        LOGGER.error("{} ...", msg, e3);
```
Comment (nisargthakkar) [inline clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java:528]: Actually, can we just skip this variable and the part of Log4j substitution? There's no "variable" anymore

File: clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java [hunk 3]
```diff
@@ -509,24 +509,23 @@ public static <T, K> T tryToDeserializeWithVerboseLogging(
...
         MD5Digest digest = new MD5Digest();
         byte[] valueChecksum = new byte[digest.getDigestSize()];
         digest.update(data.array(), data.position(), data.limit() - data.position());
         digest.doFinal(valueChecksum, 0);
-        checksumHex = Hex.encodeHexString(valueChecksum);
       } catch (Exception e2) {
-        checksumHex = "failed to compute value checksum";
-        LOGGER.error("{} ...", checksumHex, e2);
+        String msg = "failed to compute value checksum";
+        LOGGER.error("{} ...", msg, e2);
```
Comment (nisargthakkar) [inline clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java:521]: Same for this

Comment (nisargthakkar) [reply clients/venice-thin-client/src/main/java/com/linkedin/venice/client/store/AbstractAvroStoreClient.java:520]: Seems this got missed. Can you take a look again?

**Review by nisargthakkar — APPROVED** [review]

**Pull request merged**